packages/webapp/__tests__/AccountNotificationsPage.tsx

```diff
@@ -131,20 +131,6 @@ const renderComponent = (user = defaultLoggedUser): RenderResult => {
   );
 };
 
-it('should change user push notification', async () => {
```

Comment:
This won't work anymore since we are basing our subscription state on OneSignal's response and it needs integration.

Reply:
Maybe we can mock it? But yeah, probably better of as e2e backlog test once we have it.